esp/public/media/theme_editor/theme_editor.js

```diff
@@ -54,40 +54,47 @@ $j(document).ready(function(){
     showColor();
     showBasePalette();
     showCustomPalette();
-    $j('#addToPalette').click(function(){
-        var newColor = $j('<input>');
-        newColor.addClass('palette');
-        newColor.attr({
-            name: 'palette',
-            type: 'text',
-            value: 'black'
-        });
-        $j('#palette_custom_div').append(newColor);
-        $j('#palette_custom_div').append(" ");
-        showCustomPalette();
-    });
```

Comment:
Is there a particular reason that you removed this? Removing this breaks the ability to make a custom palette.